builder/vmware/common/driver_fusion6.go

```diff
@@ -124,6 +125,24 @@ func (d *Fusion6Driver) Verify() error {
 	return compareVersions(matches[1], VMWARE_FUSION_VERSION, "Fusion Professional")
 }
 
+func (d *Fusion6Driver) ToolsIsoPath(k string) string {
+	// Fusion 13.x.x changes tools iso location
+	vmxpath := filepath.Join(d.AppPath, "Contents", "Library", "vmware-vmx")
+	var stderr bytes.Buffer
+	cmd := exec.Command(vmxpath, "-v")
+	cmd.Stderr = &stderr
```

Comment:
Upon further review it looks like the command is never executed so matches which is tested below is always empty. Something like the following would solve this 
```go
       if err := cmd.Run(); err != nil {
               log.Printf("[DEBUG] failed to execute vmware-vmx command to get version %v", err)
               log.Printf("[DEBUG] continuing with default iso path for fusion6+.")
               return filepath.Join(d.AppPath, "Contents", "Library", "isoimages", "x86_64", k+".iso")
       }
```

Reply:
Thanks, added this to code.